packages/bubble-dev/start-preset/src/build.ts

```diff
@@ -48,6 +48,7 @@ export const buildReactNative = async (dir: string): Promise<StartPlugin<{}, {}>
   const allFiles = await globby(
     [
       `${dir}/src/**/*.{js,jsx,ts,tsx}`,
+      `!${dir}/src/**/*.node.{js,jsx,ts,tsx}`,
```

Comment:
This is making the react native build ignore files with `.node.(js|jsx|ts|tsx)` extension